src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -55,13 +55,26 @@ const StyledSelect = styled(TextField)`
   }
 `;
 
+const StyledLicenseImage = styled.img`
+  width: 40%;
+`;
+
+const StyledLicenseOptionImage = styled.img`
+  width: 70%;
+`;
+
 const StyledLicenseName = styled(Label)`
   margin-left: 0.5rem;
 `;
 
+const StyledLicenseOptionName = styled(Label)`
+  margin-left: -1.5rem;
```

Comment:
negativ margin? tappert :D

Reply:
Ikke superfornøyd med den nei, men tenker det er bedre enn å lage to svger av samme ikon med forskjellig viewBox